build.js

```diff
@@ -472,7 +472,10 @@ function makeAndroidProject(api, app, config, opts) {
         ]);
     })
     .then(function () {
-      var dexDir = '\nout.dexed.absolute.dir=../.dex/\nsource.dir=src\n';
+      var install_store = app.manifest.android.install_store,
+        jumbo_txt = (install_store === 'kindle' || install_store === 'underground') ?
+          'dex.force.jumbo=true\n' : '',
+        dexDir = '\nout.dexed.absolute.dir=../.dex/\nsource.dir=src\n' + jumbo_txt;
```

Comment:
Let's change it to a key that is passed from manifest like
```dex.jumbo = true```

and based on that let's change this code to add it

So only projects that absolutely need it will add it and all kindle and underground projects don't necessarily need this.